rcl_lifecycle/include/rcl_lifecycle/rcl_lifecycle.h

```diff
@@ -228,6 +228,8 @@ rcl_lifecycle_get_zero_initialized_state_machine();
  *    available transitions
  * \param[in] ts_srv_get_transition_graph pointer to the service that allows to get transitions from
  *    the graph
+ * \param[in] enable_com_interface if `false` the services listed above are not initialized,
```

Comment:
this tells me publication will be available on topic name `/<node_name>/transition_event`, but actually this publication will not be initialized with `rcl_lifecycle_com_interface_init` if enable_com_interface is false.